CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 ## [Unreleased][unreleased]
 
+## [0.11.0rc2] - 2017/07/26
+
+This new RC addresses a few performance regressions spotted in 0.11.0rc1, and
+bumps the compatible versions of OpenResty up to the latest `1.11.2.4`.
+
+It also includes support for regexps URIs and a new `kong prepare` command.
```

Comment:
"regexps" is a correct term (I personally find it less awkward in the mouth in its plural form than "regexes")